charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.75.0
+version: 3.75.1
```

Comment:
for this change we should bump the `minor` version instead of the `patch` version